src/main/check.cpp

```diff
@@ -107,16 +111,23 @@ int main(int argc, char** argv) {
     }
 
     if (domain == "zoneCrab") {
+        // Convert the instruction sequence to a control-flow graph.
```

Comment:
... in a "passive", nondeterministic form.

Reply:
Updated